packages/vite-plugin-lib/src/index.ts

```diff
@@ -195,7 +199,7 @@ function formatToFileName(entry: string, format: string): string {
 
 export function library(options: Options): Plugin[] {
   return [
-    tsconfigPaths(),
+    tsconfigPaths(options),
     buildConfig(options),
     dts({
```

Comment:
Theres actually an issue with this in dts plugin because of api extractor.

https://github.com/microsoft/rushstack/issues/4214

https://github.com/microsoft/rushstack/issues/3116

Reply:
The tool support for duplicate `d.ts` files just doesn't seem to be quite there yet (or even really required).